### PR TITLE
Sponsorship

### PR DESCRIPTION
**Pull request by debanshu08**
Made the Our sponsortship and Open Source Section.

### CODE REVIEW

File: assets/css/contact.css [hunk 1]
```diff
@@ -2,7 +2,9 @@
 /*....illustration and content....*/
 
 /** bymeacoffee**/
-
+body{
+    font-family: Montserat;
```
Comment (anushbhatia) [inline assets/css/contact.css:6]: add quotes on this then it will work

Comment (debanshu08) [reply assets/css/contact.css]: It was working on my forked local version. I have changed and added quotes though as asked for.

**Comment by anushbhatia** [comment]
[Try using this in contact.html](https://fonts.google.com/specimen/Montserrat?selection.family=Montserrat&sidebar.open=true)

File: assets/css/contact.css [hunk 2]
```diff
@@ -2,7 +2,9 @@
 /*....illustration and content....*/
 
 /** bymeacoffee**/
-
+body{
+    font-family: 'Montserat';
```
Comment (anushbhatia) [inline assets/css/contact.css:6]: font-family: 'Montserrat', sans-serif;

**Comment by anushbhatia** [comment]
resolve the merge conflict and try figuring the font issue

**Pull request merged**